test/flags-version.js

```diff
@@ -42,4 +43,20 @@ describe('flag: --version', function() {
     }
   });
 
+  it('should print only CLI version when gulp is not found', function(done) {
+    runner({ verbose: false })
+      .gulp('--version', '--cwd', os.tmpdir())
+      .run(cb);
+
+    function cb(err, stdout, stderr) {
+      expect(err).toEqual(null);
+      expect(stderr).toEqual('');
+      expect(stdout).toEqual(
+        'CLI version ' + cliVersion + '\n' +
+        ''
```

Comment:
Should this instead print `Local version: Unknown` instead of just being blank?

Reply:
I modified to print `Local version: Unknown`. And I remove the check if `env.modulePackage` is not nullish, because js-liftoff convert it from `null` to `{}` at [this line](https://github.com/js-cli/js-liftoff/blob/master/index.js#L138).

Reply:
@sttk haha, I didn't know that!